Tests/SwiftTestCodecovLibTests/AggregateTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AggregateTests.swift
+//  
+//
+//  Created by Eric DeLabar on 12/9/22.
+//
+
+import XCTest
+import Regex
+@testable import SwiftTestCodecovLib
+
+final class AggregateTests: XCTestCase {
```

Comment:
👏 .

The irony of a test coverage tool without test coverage was perhaps starting to build :)

Reply:
Lol, I didn't realize you can't actually unit test a swift package command line tool, only the library. IMHO, a weird limitation. But for whatever reason I can't get the app to work on its own code coverage, maybe for the same reason?

Reply:
Yeah, there are some unfortunate limitations there. I'm just happy you added the first actual test case to the library.